packages/devextreme/js/__internal/ui/drop_down_editor/m_drop_down_editor.ts

```diff
@@ -925,30 +1031,30 @@ const DropDownEditor = TextBox.inherit({
         this._renderSubmitElement();
         break;
       case 'rtlEnabled':
-        this._updatePopupPosition(args.value);
-        this.callBase(args);
+        this._updatePopupPosition(value as TProperties['rtlEnabled']);
+        super._optionChanged(args);
         break;
       default:
-        this.callBase(args);
+        super._optionChanged(args);
     }
-  },
+  }
 
-  open() {
+  open(): void {
     this.option('opened', true);
-  },
+  }
 
-  close() {
+  close(): void {
     this.option('opened', false);
-  },
+  }
 
-  field() {
+  field(): Element {
     return getPublicElement(this._input());
-  },
+  }
 
-  content() {
+  content(): Element | null {
     return this._popup ? this._popup.content() : null;
-  },
-});
+  }
+}
 
 registerComponent('dxDropDownEditor', DropDownEditor);
 
```

Comment:
Why do we still use export at the end of the file when this can be shortened by writing export before the class definition?

export default DropDownEditor; ––> export default class DropDownEditor { ... }

I think we can do this where we don't use the include method for mixins. Am I correct in understanding that we eventually want to get rid of the include method?

Reply:
Now, there is no mechanism to exclude Mixins without reworking the code logic, so the mixin will continue to be used for some time. The card for changing its usage was created more than two years ago, and it is not part of the task related to typing directly.

```suggestion
export default DropDownEditor; ––> export default class DropDownEditor { ... }
```
I have no objections if the code owners decide to use this notation and change it in all places at once.